tools/util.py

```diff
@@ -376,11 +378,13 @@ def mkdtemp():
 class DenoTestCase(unittest.TestCase):
     @property
     def build_dir(self):
+        # TODO(bartlomieju): cache
         args = test_args()
```

Comment:
Perhaps it suffices to do:
```
@functools.lru_cache()
def test_args():
   ...
```
I had thought that might be too clever, but perhaps it's not so bad.

Reply:
I think it's not available in Python2

Reply:
You're right, you can do something like:
```py
def test_args(argv=None, cache={}):
  argv = tuple(argv) if argv else None
  if argv in cache:
    return cache[argv]
  ...
  cache[argv] = res
  return res
```
or... something.

I think there's a clean path here that we're close to.